Assets/PatchKit Patcher/Editor/patcher.template.sh

```diff
@@ -1,19 +1,51 @@
 #!/usr/bin/env sh
 
-EXEDIR=$1
-PATCHER_EXE=$2
-SECRET=$3
-INSTALLDIR=$4
-LOCKFILE=$5
+EXEDIR="."
+PATCHER_EXE="Patcher"
+LOCKFILE=""
+NETWORK_STATUS="online"
```

Comment:
Hm, can we crash this script in case of not providing those values, instead of setting some default ones?